apps/src/convert.cpp

```diff
@@ -1,83 +1,72 @@
 #include <iostream>
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
 #include <opencv2/core.hpp>
 #include <opencv2/imgcodecs.hpp>
 #include <opencv2/photo.hpp>
 
-#include <boost/filesystem.hpp>
+#include "envitools/EnviUtils.hpp"
 
 namespace fs = boost::filesystem;
+namespace po = boost::program_options;
+namespace et = envitools;
 
 constexpr static uint16_t MAX_INTENSITY = std::numeric_limits<uint16_t>::max();
```

Comment:
Make this MAX_INTENSITY_16BPC. Also add a similar one for MAX_INTENSITY_8BPC (the template type for numeric_limits will be uint8_t)